### ISSUE TITLE
Eclipse/Flow mismatch --> Producer well closed in Flow

### ISSUE THREAD

**Issue by andrescodas** [OPM/opm-simulators#1688]
Consider the example attached.  For some reason Flow does not produce from well PRODUCER. The test was performed on Flow compiled from sources just today.

[deck_closed_producer.zip](https://github.com/OPM/opm-simulators/files/2704460/deck_closed_producer.zip)
![injector](https://user-images.githubusercontent.com/164282/50366427-5c946200-0558-11e9-91f1-dd770912114e.png)
![producer](https://user-images.githubusercontent.com/164282/50366431-5f8f5280-0558-11e9-9da5-c24814cb6913.png)

**Comment by GitPaean** [comment]
Thanks for the example. It is something we will need to work on. The reason is that the BHP of the producers is higher than the reservoir, so it could not produce. 

Somehow, ECL manages to solve it. There will be some adjustments to our solution strategy to handle this type of situation.

**Comment by andrescodas** [comment]
Hi Kai.  I repeated the experiment with a lower BHP pressure for the producer.  Further, you see the pressure from the injector increasing rapidly to the default limit. Thus, I believe you are not right about " the BHP of the producers is higher than the reservoir, so it could not produce".  This example is setting explicitly the transmissibilities of all grid-blocks, transmissibilities of the wells, pore-volumes, and depth of the grid-block centers.  I wonder if the issue could be related to any of these features.


![injector](https://user-images.githubusercontent.com/164282/50374437-42e13200-05cd-11e9-95c4-e1939a85c98c.png)
![producer](https://user-images.githubusercontent.com/164282/50374440-44aaf580-05cd-11e9-9efd-f153054f01a6.png)

**Comment by GitPaean** [comment]
Yes. I can be wrong. Let us continue the discussion after new year.

Outlook<https://aka.ms/qtex0l> for iOS

________________________________

**Comment by GitPaean** [comment]
Hi, something interesting is that, ECL closes the injector well, while opm keeps it open (OPM has not provided many strategy to close injector yet). 

And ECL keeps producer open, while OPM closes the producer.

**Comment by andrescodas** [comment]
> ECL closes the injector well, while opm keeps it open
At what point ECL closes the injector?.  From the figures I see both wells always open for ECL. It is OPM which closes the injector after the reservoir reaches a pressure limit. ECL injects constantly at 10 m^3/day.  Further, for OPM there is a marginal production in the first time step.

**Comment by GitPaean** [comment]
> At what point ECL closes the injector?

I misread it. I saw the red line BHP so low (relatively), and qOs is zero, misread qWs to be zero too.  Sorry about that. I was surprised that I did not find any related message.  

OPM did not shut injector, it is just very small injection rate.
